### PR TITLE
Compute velocity of MPCD particles

### PR DESCRIPTION
**Pull request by mphoward**
This PR adds a CPU C++ compute for the net velocity of MPCD particles. It is based off PR #64 and should be merged after that one.

### CODE REVIEW

**Comment by mphoward** [comment]
I compared this to a Python implementation on stampede2, and the results were identical.

File: azplugins/analyze.py [hunk 1]
```diff
@@ -18,6 +21,58 @@
...
+    ``vy``, and ``vz`` with ``suffix`` appended. By default, ``suffix`` is
+    ``_name`` where ``name`` is the name of the ``group``, like ``_all`` for
+    :py:class:`hoomd.group.all`. However, a custom suffix may be specified; the
+    only requirement is that the same suffix cannot be used more than once. You
+    can save these results using :py:class:`hoomd.analyze.log`.
+
+    Example::
+
+        azplugins.analyze.group_velocity(hoomd.group.all())
+        hoomd.analze.log(filename='velocity.dat', quantities=['vx_all'], period=10)
```
Comment (astatt) [inline azplugins/analyze.py:50]: Could you add an example with a custom name? Just to be overly explicit for users that are less experienced with the logger? 

Also, there is a typo "analze" -> "analyze".

File: azplugins/mpcd.py [hunk 2]
```diff
@@ -23,6 +25,54 @@
...
+    The components of the result are exposed as loggable quantities ``mpcd_vx``,
+    ``mpcd_vy``, and ``mpcd_vz`` with ``suffix`` appended. By default,
+    ``suffix`` is an empty string, but a custom suffix may be specified if needed
+    to distinguish the logged quantity from something else. You can save these
+    results using :py:class:`hoomd.analyze.log`.
+
+    Example::
+
+        azplugins.mpcd.compute_velocity()
+        hoomd.analyze.log(filename='velocity.dat', quantities=['mpcd_vx'], period=10)
```
Comment (astatt) [inline azplugins/mpcd.py:53]: same comment as above, maybe add an example with a custom name just for the sake of completeness.

**Review by astatt — CHANGES_REQUESTED** [review]
Looks very good! I have very very minor comments, you can decide of you want to address them or not. In any case, this is ready to merge. I marked as "request changes" such that you can look at my comments. 

The only one you need to address is the spelling mistake :), everything else is totally optional.

File: azplugins/test-py/test_analyze_group_velocity.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    def setUp(self):
+        snap = hoomd.data.make_snapshot(N=2, box=hoomd.data.boxdim(Lx=10,Ly=10,Lz=10), particle_types=['A','B'])
+        if hoomd.comm.get_rank() == 0:
+            snap.particles.position[:] = [[0,0,-5],[0,0,5]]
+            snap.particles.velocity[:] = [[1,-2,3],[-2,4,-6]]
+            snap.particles.typeid[:] = [0,1]
+            snap.particles.mass[:] = [1,2]
+
+        self.s = hoomd.init.read_snapshot(snap)
+
```
[the remark this answers is not in the document]
Comment (mphoward) [reply azplugins/test-py/test_analyze_group_velocity.py:25]: Good idea, test added on PR #64

File: azplugins/analyze.py (hunk 1 above)
Comment (mphoward) [reply azplugins/analyze.py:50]: Resolved on #64

**Comment by mphoward** [comment]
Should be taken care of! Can you hit approve so I can see if it will let me merge without using my admin privilege?

**Review by astatt — APPROVED** [review]
Good to go!

**Pull request merged**